source/agora/node/Runner.d

```diff
@@ -237,9 +250,11 @@ private TLSContext getTLSContext ()
         const cert_path = cert_search_paths[idx] ~ cert_file;
         const key_path = key_search_paths[idx] ~ key_file;
 
+        log.info("Checking for {} and {}", cert_path, key_path);
         // Enable TLS
         if (exists(cert_path) && exists(key_path))
         {
+            log.info("Found tls files at {} and {}", cert_path, key_path);
```

Comment:
Likewise: http://www.linfo.org/rule_of_silence.html#:~:text=The%20Rule%20of%20Silence,say%2C%20it%20should%20say%20nothing.&text=That%20is%2C%20silence%20in%20itself%20is%20a%20virtue.

Reply:
Removed